### PR TITLE
Color consolidation into variables

### LINKED ISSUE #856: Less variable refactoring (bring layout pixels and color hex values in variables)
I would like to use fuelux in my theme, but the less files don't use variable.less.
For example you define colors inside checkbox.less. Sadly there is not much point in using variable.less this way...

Are you planning to fix this?

### PR DESCRIPTION
**Pull request by interactivellama**
Removes all hex values from LESS files. There were a few grays that had minimal saturation that were converted to hue-less grays. A few grays may have changed by 1%-3% luminance in order to create consistency. There are still 20+ grays being used.

Fixes #856 and makes getting to the idea of #972 easier although developers will still need to modify `fuelux/less/variables.less` if they would like to theme the controls.

### CODE REVIEW

**Comment by interactivellama** [comment]
Removed `dist` files and squashed!

File: less/checkbox.less [hunk 1]
```diff
@@ -51,10 +51,10 @@
 		position: relative;
 
 		&:before {
-			#fuelux > .icon-base-styles("Glyphicons Halflings");
```
Comment (swilliamset) [inline less/checkbox.less]: this branch needs to be rebased/merged with the mixin namespacing solution

**Comment by interactivellama** [comment]
Updated. `grunt-browserify` was not on the Heroku edge server.

**Comment by interactivellama** [comment]
@swilliamset Updated by removing edits from other PRs that are now present since merging the past few days.

**Comment by swilliamset** [comment]
We need to consider whether or not `lighten()` and `desaturate()` against a variable is recommended. Should the resulting values be equal to a variable already? Otherwise excepting the PR.

**Pull request merged**